# Timestamp
builtin_interfaces/Time ts

# Sensor values
float64 sensor1
float64 sensor2
float64 sensor3
float64 sensor4

# Motor encoders
int32 left_motor_encoder
int32 right_motor_encoder
---
int32 left_motor_speed
int32 right_motor_speed
